set.mm

```diff
@@ -503742,6 +504060,64 @@ The sumset operation can be used for both group (additive) operations and
       QCUOUJUPCUOHUKBUMUOCUREUDUEUFUGUJUNUMHUKBUMCUREUHTUI $.
   $}
 
+  ${
+    pidlnzb.1 $e |- B = ( Base ` R ) $.
+    pidlnzb.2 $e |- .0. = ( 0g ` R ) $.
+    pidlnzb.3 $e |- K = ( RSpan ` R ) $.
+    $( A principal ideal is nonzero iff is is generated by a nonzero elements
```

Comment:
"is is" -> "it is"

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4730/commits/de38c970914f120d4aa4bcc4f56ac8e5e0858f8a